prog/download_boot_image.rb

```diff
@@ -1,14 +1,25 @@
 # frozen_string_literal: true
 
+require "json"
+
 class Prog::DownloadBootImage < Prog::Base
   subject_is :sshable, :vm_host
 
   def image_name
     @image_name ||= frame.fetch("image_name")
   end
 
-  def custom_url
-    @custom_url ||= frame["custom_url"]
+  def version
+    @version ||= frame.fetch("version")
+  end
+
+  def url
+    @url ||=
+      if frame["custom_url"]
+        frame["custom_url"]
+      elsif image_name.start_with?("github")
```

Comment:
We also have custom PG images.
```suggestion
      elsif image_name.start_with?("github") || image_name.start_with?("postgres")
```

Reply:
ah, and actually we were considering using different prefixes for partnerships, such as `companyname-postgres`, but that is OK, we can switch to `postgres-companyname`